tests/frontend/onnx/test_tensor_type.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from xdsl.dialects.builtin import TensorType, f32, f64
+
+pytest.importorskip("onnx-weekly", reason="onnx is an optional dependency")
```

Comment:
Maybe I am trippin, but this kind of does not make 100% sense to me. You are including from onnx in the line below, but here you are checking whether you can include onnx-weekly? (This might explain the coverage thingi)

Reply:
Yeah this was super confusing to me. It outright fails for `"onnx"`, and skips successfully with `"onnx-weekly"`. Maybe I'm not getting something, but I don't understand how it can fail to not find a package that's not there...

Reply:
I'll try again now with a new venv